docusaurus.config.js

```diff
@@ -14,7 +14,11 @@ const docCollectionsLocation = './src/data/doc-collections';
 let cachedSources;
 
 const mixpanelOnLoad = `
+<<<<<<< HEAD
 if ('${process.env.MIXPANEL_PROJECT_TOKEN}' && '${process.env.MIXPANEL_PROJECT_TOKEN}' !== 'undefined') {
+=======
+if (${process.env.MIXPANEL_PROJECT_TOKEN}) {
+>>>>>>> 093b6c85a (add hotjar")
```

Comment:
this is a merge conflict
```suggestion
if ('${process.env.MIXPANEL_PROJECT_TOKEN}' && '${process.env.MIXPANEL_PROJECT_TOKEN}' !== 'undefined') {
```